### PR TITLE
hogaeng.co.kr - Ad

### PR DESCRIPTION
**Pull request by green1052**

### CODE REVIEW

File: filters-share/specific_ELEMHIDE.txt [hunk 1]
```diff
@@ -59,6 +59,9 @@ sports.donga.com##div[id*="_AD"]
 ! ##div[id$="_Ad"]
 yes24.com##div[id$="_Ad"]
 !
+hogaeng.co.kr###partnership
+hogaeng.co.kr##.sponsor
+hogaeng.co.kr##.sponsor2
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: Can be compressed.

File: filters-uBO/extended_css_ELEMHIDE.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+hogaeng.co.kr#?#.bo_notice:has(.spon_b_icon1)
```
Comment (piquark6046) [inline filters-uBO/extended_css_ELEMHIDE.txt]: Can be merged to `filters-share/extended_css_ELEMHIDE.txt`.
No such elements exist.

File: filters-AG/extended_css_ELEMHIDE.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
+hogaeng.co.kr#?#.bo_notice:has(> .spon_b_icon1)
```
Comment (piquark6046) [inline filters-AG/extended_css_ELEMHIDE.txt]: Can be merged to `filters-share/extended_css_ELEMHIDE.txt`.
No such elements exist.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 1 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: like hogaeng.co.kr##div[class^="sponsor"] ?

Comment (piquark6046) [reply filters-share/specific_ELEMHIDE.txt]: Yes.

Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: resloved pls review again

File: filters-share/specific_ELEMHIDE.txt [hunk 4]
```diff
@@ -59,6 +59,9 @@ sports.donga.com##div[id*="_AD"]
 ! ##div[id$="_Ad"]
 yes24.com##div[id$="_Ad"]
 !
+hogaeng.co.kr##.bo_notice:has(.spon_b_icon1)
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: Currently, Mozilla Firefox does not support `:has()` widely.
https://developer.mozilla.org/en-US/docs/Web/CSS/:has

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 4 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: so how to do?

Comment (piquark6046) [reply filters-share/specific_ELEMHIDE.txt]: Please move to `/filters-share/extended_css_ELEMHIDE.txt`.

Comment (piquark6046) [reply filters-share/specific_ELEMHIDE.txt]: With changing from a cosmetic filter to an extended CSS filter.

Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: sry its my mistake

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Please read https://github.com/List-KR/NamuLink/pull/10#pullrequestreview-1215935579 again.

**Comment by green1052** [comment]
i know but i cant access github web editer so i just pushed sry

**Comment by piquark6046** [comment]
Then, install Visual Studio Code on your local computer or recheck configuration of your extensions and web browser.
If you cannot that, I cannot keep your permission accessing to List-KR filter repository.

**Comment by green1052** [comment]
ok

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**